.github/workflows/release.yml

```
@@ -3,73 +3,60 @@ name: Release
-    name: Create Release
+  build-skill:
+    name: Build Claude Code Skill
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
     steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)